swap/swap_test.go

```diff
@@ -112,13 +113,29 @@ func newTestBackend() *swapTestBackend {
 	// commit the initial "pre-mined" accounts (issuer and beneficiary addresses)
 	defaultBackend.Commit()
 
+	tokenAddress, _, token, err := contractFactory.DeployERC20Mintable(bind.NewKeyedTransactor(ownerKey), defaultBackend)
+	if err != nil {
+		log.Crit(err.Error())
+	}
+	defaultBackend.Commit()
+
+	_, err = token.Mint(bind.NewKeyedTransactor(ownerKey), ownerAddress, big.NewInt(1000000000000000000))
```

Comment:
If a return param is ignored, please always add a comment why, or don't ignore it

Reply:
ok!